locales/pt-BR/deliverytool.rdf.po

```diff
@@ -1,21 +1,14 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: tao-3x\n"
-"PO-Revision-Date: 2023-04-21 15:46\n"
+"Project-Id-Version: TAO 2023.12\n"
+"PO-Revision-Date: 2023-11-30T14:56:09\n"
 "Last-Translator: TAO Translation Team <translation@tao.lu>\n"
 "MIME-Version: 1.0\n"
 "Language: pt-BR\n"
 "sourceLanguage: en-US\n"
 "targetLanguage: pt-BR\n"
-"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Crowdin-Project: tao-3x\n"
-"X-Crowdin-Project-ID: 582145\n"
-"X-Crowdin-Language: pt-BR\n"
-"X-Crowdin-File: /ltiDeliveryProvider/locales/en-US/deliverytool.rdf.po\n"
-"X-Crowdin-File-ID: 1219\n"
-"Language-Team: Portuguese, Brazilian\n"
```

Comment:
**thought:** I don't know to which extent removing these metadata may impact the link with Crowdin. It may be worth keeping them.

Reply:
Done: https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/367/commits/b7ad5e2b5703607dee87184dbcefd4f5533daa2f